.gitmodules

```diff
@@ -87,6 +87,10 @@
 	path = sources/percona-images
 	url = https://github.com/Percona-Lab/percona-images
 	branch = master
+[submodule "aws-iam-authenticator"]
+	path = sources/aws-iam-authenticator/src/github.com/kubernetes-sigs/aws-iam-authenticator
+	url = https://github.com/kubernetes-sigs/aws-iam-authenticator
+	branch = v0.5.1
```

Comment:
We use submodules only for repositories on Percona accounts

(clickhouse_exporter is an exception, but upstream is frozen and actually refers to our fork now)